# Crimean Turkish; Crimean Tatar translation for update-manager
# Copyright (c) 2010 Rosetta Contributors and Canonical Ltd 2010
# This file is distributed under the same license as the update-manager package.
#
# Reşat SABIQ <tilde.birlik@gmail.com>, 2010, 2011.
#: ../DistUpgrade/DistUpgradeController.py:1076
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2020-05-05 15:30-0700\n"
"PO-Revision-Date: 2013-05-22 11:00+0000\n"
"Last-Translator: Reşat SABIQ <tilde.birlik@gmail.com>\n"
"Language-Team: QIRIMTATARCA (Qırım Türkçesi) <tilde-birlik-tercime@lists."
"sourceforge.net>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Launchpad-Export-Date: 2019-11-15 04:46+0000\n"
"X-Generator: Launchpad (build c597c3229eb023b1e626162d5947141bf7befb13)\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:215 ../DistUpgrade/distro.py:446
#, python-format
msgid "Server for %s"
msgstr "%s içün sunucı"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:233 ../DistUpgrade/distro.py:239
#: ../DistUpgrade/distro.py:255
msgid "Main server"
msgstr "Ana Sunucı"

#: ../DistUpgrade/distro.py:259
msgid "Custom servers"
msgstr "Şahsiyleştirilgen sunucılar"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:151
msgid "Could not calculate sources.list entry"
msgstr "sources.list kirişi esaplanamadı"

# tr
#: ../DistUpgrade/DistUpgradeAptCdrom.py:260
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Hiçbir paket dosyasının yeri belirlenemedi, belki de bu bir Ubuntu Diski "
"değildir ya da yanlış bir mimaridir?"

# tr
#: ../DistUpgrade/DistUpgradeAptCdrom.py:310
msgid "Failed to add the CD"
msgstr "CD eklemede hata"

# tr
#: ../DistUpgrade/DistUpgradeAptCdrom.py:311
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"CD eklenirken bir hata oluştu, yükseltme durduruluyor. Eğer geçerli bir "
"Ubuntu CD'si kullanıyorsanız bunu bir hata  olarak bildirin.\n"
"Hata iletisi:'%s'"

# tr
#: ../DistUpgrade/DistUpgradeCache.py:180
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Hatali durumdaki paket(ler)i kaldır"

# tr
#: ../DistUpgrade/DistUpgradeCache.py:183
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"%s paket(ler)i tutarsız bir durumda ve yeniden yüklenmesi gerekiyor fakat bu "
"paket(ler) için uygun arşiv dosyaları bulunamadı. Devam etmek için bu "
"paket(ler)i şimdi silmek ister misiniz?"

# tr
#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:299
msgid "The server may be overloaded"
msgstr "Server'a fazla yüklenmiş olabilir"

#: ../DistUpgrade/DistUpgradeCache.py:412
msgid "Broken packages"
msgstr "Sınıq paketler"

# tr
#: ../DistUpgrade/DistUpgradeCache.py:413
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Sisteminiz bu yazılım ile düzeltilemeyen bozuk paketler içeriyor. Devam "
"etmeden önce lütfen bunları synaptic veya apt-get kullanarak düzeltin."

#: ../DistUpgrade/DistUpgradeCache.py:649
#, fuzzy, python-format
msgid "Broken packages after upgrade: %s"
msgstr "Sınıq paketler"

#. the most likely problem is the 3rd party pkgs so don't address
#. foreignPkgs and devRelease being True
#: ../DistUpgrade/DistUpgradeCache.py:658
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:661
msgid ""
"This was likely caused by:\n"
" * Unofficial software packages not provided by Ubuntu\n"
"Please use the tool 'ppa-purge' from the ppa-purge \n"
"package to remove software from a Launchpad PPA and \n"
"try the upgrade again.\n"
"\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:669
msgid ""
"This was caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
"This is most likely a transient problem, \n"
"please try again later.\n"
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeCache.py:676
msgid "This is most likely a transient problem, please try again later."
msgstr ""
"Bu geçici bir problem gibi gözüküyor, lütfen daha sonra tekrar deneyiniz."

#: ../DistUpgrade/DistUpgradeCache.py:679
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal. "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:681
msgid ""
"If you want to investigate this yourself the log files in '/var/log/dist-"
"upgrade' will contain details about the upgrade. Specifically, look at 'main."
"log' and 'apt.log'."
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:687
msgid "Could not calculate the upgrade"
msgstr "Üst-qademeleme esaplanamadı"

# tr
#: ../DistUpgrade/DistUpgradeCache.py:746
msgid "Error authenticating some packages"
msgstr "Bazı paketlerin doğrulamasında hata oluştu"

# tr
#: ../DistUpgrade/DistUpgradeCache.py:747
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Bazı paketler doğrulanamadı. Bu geçici bir ağ sorunu olabilir. Daha sonra "
"tekrar deneyebilirsiniz. Doğrulanamamış paketlerin listesi için aşağıya "
"bakınız."

# tr
#: ../DistUpgrade/DistUpgradeCache.py:768
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"'%s' paketi kaldırılması için işaretlenmiş ancak paket kaldırma kara listesi "
"içinde."

# tr
#: ../DistUpgrade/DistUpgradeCache.py:774
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Temel '%s' paketi kaldırılma için işaretlenmiş."

# tr
#: ../DistUpgrade/DistUpgradeCache.py:783
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Kara listede olan '%s' sürümü yüklenmeye çalışılıyor"

# tr
#: ../DistUpgrade/DistUpgradeCache.py:905
#, python-format
msgid "Can't install '%s'"
msgstr "\"%s\" yüklenemiyor"

#: ../DistUpgrade/DistUpgradeCache.py:906
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeCache.py:917
msgid "Can't guess meta-package"
msgstr "Meta-paket tahmin edilemiyor"

#: ../DistUpgrade/DistUpgradeCache.py:918
#, python-format
msgid ""
"Your system does not contain a %s or %s package and it was not possible to "
"detect which version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:113
msgid "Reading cache"
msgstr "Öghafiza oqula"

# tr
#: ../DistUpgrade/DistUpgradeController.py:226
msgid "Unable to get exclusive lock"
msgstr "Özel kilide erişilemiyor"

# tr
#: ../DistUpgrade/DistUpgradeController.py:227
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Bu, genelde apt-get ya da aptitude gibi başka bir paket yönetimi "
"uygulamasının çalıştığı anlamına gelir. Lütfen öncelikle bu uygulamayı "
"kapatın."

# tr
#: ../DistUpgrade/DistUpgradeController.py:285
msgid "Upgrading over remote connection not supported"
msgstr "Uzaktan bağlantı ile yükseltme desteklenmiyor"

# tr
#: ../DistUpgrade/DistUpgradeController.py:286
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Bir uzak ssh bağlantısı üzerinden bunu desteklemeyen bir ön uç ile yükseltme "
"işlemini gerçekleştiriyorsunuz. Lütfen 'do-release-upgrade' komutu ile "
"beraber metin tabanlı bir yükseltme deneyin.\n"
"Yükseltme işlemi iptal edilecek. Lütfen ssh kullanmadan tekrar deneyin."

# tr
#: ../DistUpgrade/DistUpgradeController.py:300
msgid "Continue running under SSH?"
msgstr "SSH altında çalışmaya devam edilsin mi?"

# tr
#: ../DistUpgrade/DistUpgradeController.py:301
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Bu oturum ssh altında çalışır halde görünüyor. Bir hata oluşması durumunda "
"düzeltilmesinin zor olması nedeniyle, yükseltmeyi ssh üzerinden "
"gerçekleştirmeniz tavsiye edilmez.\n"
"\n"
"Devam ederseniz, ek bir ssh art hizmeti '%s' bağlantı noktasında "
"başlatılacak.\n"
"Devam etmek istiyor musunuz?"

# tr
#: ../DistUpgrade/DistUpgradeController.py:315
msgid "Starting additional sshd"
msgstr "Yeni bir sshd başlatılıyor"

# tr
#: ../DistUpgrade/DistUpgradeController.py:316
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Hata durumunda düzeltme daha kolay olsun diye ek bir sshd '%s' portunda "
"başlatılacak. Eğer çalışmakta olan ssh ile ilgili herhangi bir sorun "
"oluşursa ek birine bağlanabilirsiniz.\n"

#: ../DistUpgrade/DistUpgradeController.py:324
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Eger bir ateş-divarı qullana iseñiz, bu limannı muvaqqat olaraq açmañız "
"kerekebilir. Potensial olaraq telükeli olğanından dolayı, bu avtomatik "
"olaraq yapılmay. Limannı meselâ bunıñ ile açabilirsiñiz:\n"
"'%s'"

# tr
#: ../DistUpgrade/DistUpgradeController.py:402
#: ../DistUpgrade/DistUpgradeController.py:426
msgid "Can not upgrade"
msgstr "Yükseltilemiyor"

# tr
#: ../DistUpgrade/DistUpgradeController.py:403
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Bu araçla, '%s' sürümünden '%s' sürümüne bir yükseltme desteklenmiyor."

#: ../DistUpgrade/DistUpgradeController.py:427
msgid ""
"Your python3 install is corrupted. Please fix the '/usr/bin/python3' symlink."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:456
#, python-format
msgid "Can not write to '%s'"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:457
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:468
msgid "Include latest updates from the Internet?"
msgstr "İnternetten en soñki yañartmalar da dahil etilsinmi?"

# tr
#: ../DistUpgrade/DistUpgradeController.py:469
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Güncelleme sistemi interneti kullanarak en son güncellemeleri otomatik "
"olarak indirebilir ve yükseltme sırasında güncellemeleri kurabilir. Eğer bir "
"ağ bağlantınız varsa şiddetle önerilir.\n"
"\n"
"Güncelleme uzun sürebilir fakat tamamlandığında sisteminiz tamamıyla güncel "
"olacak. Güncelleştirmeleri şu anda yapmayabilirsiniz ama güncelleştirmeyi "
"sisteminizi yükselttikten hemen sonra yapmanız tavsiye edilir.\n"
"Eğer cevabınız 'hayır' ise , ağ bağlantısı kullanılmadan devam edilecek."

#: ../DistUpgrade/DistUpgradeController.py:647
#, python-format
msgid "Not for humans during development stage of release %s"
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeController.py:748
#, python-format
msgid "disabled on upgrade to %s"
msgstr "%s güncelleştirilmesi iptal edildi."

#: ../DistUpgrade/DistUpgradeController.py:805
msgid "No valid sources.list entry found"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:806
#, python-format
msgid ""
"While scanning your repository information no entry about %s could be "
"found.\n"
"\n"
"An upgrade might not succeed.\n"
"\n"
"Do you want to continue anyway?"
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeController.py:818
msgid "No valid mirror found"
msgstr "Geçerli yansı bulunamadı"

#: ../DistUpgrade/DistUpgradeController.py:819
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run an internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""

# tr
#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:839
msgid "Generate default sources?"
msgstr "Öntanımlı depolar oluşturulsun mu?"

# tr
#: ../DistUpgrade/DistUpgradeController.py:840
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"'sources.list' dosyanız tarandıktan sonra '%s' için geçerli bir kayıt "
"bulunamadı.\n"
"'%s' için öntanımlı kayıtlar eklensin mi? Eğer 'Hayır' seçerseniz, yükseltme "
"iptal edilecek."

# tr
#: ../DistUpgrade/DistUpgradeController.py:875
#: ../DistUpgrade/DistUpgradeController.py:881
msgid "Repository information invalid"
msgstr "Depo bilgisi geçersiz"

#: ../DistUpgrade/DistUpgradeController.py:876
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:882
msgid ""
"Upgrading the repository information resulted in a invalid file. To report a "
"bug install apport and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeController.py:891
msgid "Third party sources disabled"
msgstr "Üçüncü taraf kaynaklar devre dışı bırakıldı"

# tr
#: ../DistUpgrade/DistUpgradeController.py:892
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"sources.list dosyasındaki bazı üçüncü taraf girdiler etkisiz hale getirildi. "
"Yükseltmenin ardından bu girdileri 'Yazılım Kaynakları' aracını ya da paket "
"yöneticinizi kullanarak yeniden etkinleştirebilirsiniz."

# tr
#: ../DistUpgrade/DistUpgradeController.py:934
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Paket tutarsız durumda"

# tr
#: ../DistUpgrade/DistUpgradeController.py:937
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"'%s' paketi tutarsız durumda ve tekrar kurulmalı, fakat bunun için herhangi "
"bir arşiv bulunamadı. Lütfen paketi elle tekrar kurun ya da sistemden "
"kaldırın."

# tr
#: ../DistUpgrade/DistUpgradeController.py:1000
msgid "Error during update"
msgstr "Güncelleme sırasında hata"

# tr
#: ../DistUpgrade/DistUpgradeController.py:1001
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Güncelleştirme sırasında bir hata oluştu. Bu genellikle bir ağ sorunudur, "
"lütfen ağ bağlantınızı denetleyin ve tekrar deneyin."

#: ../DistUpgrade/DistUpgradeController.py:1044
#: ../DistUpgrade/DistUpgradeController.py:1049
msgid "EFI System Partition (ESP) not usable"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1045
msgid ""
"Your EFI System Partition (ESP) is not mounted at /boot/efi. Please ensure "
"that it is properly configured and try again."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1050
msgid ""
"The EFI System Partition (ESP) mounted at /boot/efi is not writable. Please "
"mount this partition read-write and try again."
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeController.py:1057
msgid "Not enough free disk space"
msgstr "Yeterince boş disk alanı yok"

#. TRANSLATORS: you can change the order of the sentence,
#. make sure to keep all {str_*} string untranslated.
#: ../DistUpgrade/DistUpgradeController.py:1060
#, python-brace-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of {str_total} free space "
"on disk '{str_dir}'. Please free at least an additional {str_needed} of disk "
"space on '{str_dir}'. {str_remedy}"
msgstr ""

#. specific ways to resolve lack of free space
#: ../DistUpgrade/DistUpgradeController.py:1065
msgid ""
"Remove temporary packages of former installations using 'sudo apt clean'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1067
msgid ""
"You can remove old kernels using 'sudo apt autoremove' and you could also "
"set COMPRESS=xz in /etc/initramfs-tools/initramfs.conf to reduce the size of "
"your initramfs."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1072
msgid ""
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1075
msgid "Reboot to clean up files in /tmp."
msgstr ""

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:1114
#: ../DistUpgrade/DistUpgradeController.py:1990
#: ../DistUpgrade/DistUpgradeController.py:2040
msgid "Calculating the changes"
msgstr "Deñişiklikler esaplana"

# tr
#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:1158
msgid "Do you want to start the upgrade?"
msgstr "Yükseltmeyi başlatmak istiyor musunuz?"

#: ../DistUpgrade/DistUpgradeController.py:1177
#, python-format
msgid ""
"Livepatch security updates are not available for Ubuntu %s. If you upgrade, "
"Livepatch will turn off."
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeController.py:1249
msgid "Upgrade canceled"
msgstr "Yükseltme iptal edildi"

#: ../DistUpgrade/DistUpgradeController.py:1250
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Üst-qademeleme şimdi lâğu etecek ve özgün sistem durumı keri tiklenecek. Üst-"
"qademelemeni soñraki bir zamanda devam etebilirsiñiz."

# tr
#: ../DistUpgrade/DistUpgradeController.py:1256
#: ../DistUpgrade/DistUpgradeController.py:1401
msgid "Could not download the upgrades"
msgstr "Yükseltmeler indirilemedi"

#: ../DistUpgrade/DistUpgradeController.py:1257
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""

# tr
#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1350
#: ../DistUpgrade/DistUpgradeController.py:1389
#: ../DistUpgrade/DistUpgradeController.py:1505
msgid "Error during commit"
msgstr "İşlem sırasında hata oluştu"

# tr
#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1352
#: ../DistUpgrade/DistUpgradeController.py:1391
#: ../DistUpgrade/DistUpgradeController.py:1544
msgid "Restoring original system state"
msgstr "Özgün sistem durumuna geri dönülüyor"

# tr
#: ../DistUpgrade/DistUpgradeController.py:1353
#: ../DistUpgrade/DistUpgradeController.py:1368
#: ../DistUpgrade/DistUpgradeController.py:1392
msgid "Could not install the upgrades"
msgstr "Yükseltmeler kurulamadı"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1358
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Üst-qademeleme abortlandı. Sistemiñiz qullanılalmaz bir durumda olabilir. "
"Şimdi bir qurtarma çaptırılacaq (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1363
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug and attach the files in /var/log/"
"dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1402
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Üst-qademeleme abortlandı. Lütfen İnternet bağlantıñıznı ya da qurulım "
"vasatıñıznı teşkeriñiz ve tekrar deñeñiz. "

# tr
#: ../DistUpgrade/DistUpgradeController.py:1430
msgid "Searching for obsolete software"
msgstr "Kullanılmayan yazılımlar aranıyor"

#: ../DistUpgrade/DistUpgradeController.py:1493
msgid "Remove obsolete packages?"
msgstr "Eskirgen paketler çetleştirilsinmi?"

#: ../DistUpgrade/DistUpgradeController.py:1494
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Qoru"

#: ../DistUpgrade/DistUpgradeController.py:1494
msgid "_Remove"
msgstr "_Çetleştir"

# tr
#: ../DistUpgrade/DistUpgradeController.py:1506
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Temizlik sırasında bazı sorunlar oluştu. Daha fazla bilgi için lütfen "
"aşağıdaki iletiye bakın. "

# tr
#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1582
msgid "Required depends is not installed"
msgstr "Gerekli bağımlılıklar kurulu değil"

# tr
#: ../DistUpgrade/DistUpgradeController.py:1583
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Gerekli bağımlılık '%s' kurulu değil. "

# tr
#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1850
#: ../DistUpgrade/DistUpgradeController.py:1940
msgid "Checking package manager"
msgstr "Paket yöneticisi denetleniyor"

#: ../DistUpgrade/DistUpgradeController.py:1856
#: ../DistUpgrade/DistUpgradeController.py:1862
msgid "Preparing the upgrade failed"
msgstr "Üst-qademeleme azırlığı muvafaqiyetsiz edi"

#: ../DistUpgrade/DistUpgradeController.py:1857
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1863
msgid ""
"Preparing the system for the upgrade failed. To report a bug install apport "
"and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeController.py:1880
#: ../DistUpgrade/DistUpgradeController.py:1890
msgid "Getting upgrade prerequisites failed"
msgstr "Güncelleme önkoşullarının alınması başarısız"

#: ../DistUpgrade/DistUpgradeController.py:1881
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1891
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"To report a bug install apport and then execute 'apport-bug ubuntu-release-"
"upgrader'."
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeController.py:1924
msgid "Updating repository information"
msgstr "Depo bilgileri güncelleniyor"

#: ../DistUpgrade/DistUpgradeController.py:1931
msgid "Failed to add the cdrom"
msgstr "Kompakt disk faqat-oqulır hazifa (cdrom) eklenamadı"

#: ../DistUpgrade/DistUpgradeController.py:1932
msgid "Sorry, adding the cdrom was not successful."
msgstr ""
"Afu etiñiz, kompakt disk faqat-oqulır hazifanıñ (cdrom) eklenmesi "
"muvafaqiyetli degil edi."

# tr
#: ../DistUpgrade/DistUpgradeController.py:1967
msgid "Invalid package information"
msgstr "Geçersiz paket bilgisi"

#: ../DistUpgrade/DistUpgradeController.py:1968
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1997
#: ../DistUpgrade/DistUpgradeController.py:2096
msgid "Fetching"
msgstr "Ketirile"

#: ../DistUpgrade/DistUpgradeController.py:2004
#: ../DistUpgrade/DistUpgradeController.py:2025
#: ../DistUpgrade/DistUpgradeController.py:2046
#: ../DistUpgrade/DistUpgradeController.py:2100
msgid "Upgrading"
msgstr "Üst-qademelene"

#: ../DistUpgrade/DistUpgradeController.py:2006
msgid "Upgrade infeasible"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2007
msgid ""
"The upgrade could not be completed, there were errors during the upgrade "
"process."
msgstr ""

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2028
msgid "Upgrade incomplete"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2029
msgid ""
"The upgrade has partially completed but there were errors during the upgrade "
"process."
msgstr ""

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2051
#: ../DistUpgrade/DistUpgradeController.py:2102
#: ../DistUpgrade/DistUpgradeController.py:2108
#: ../DistUpgrade/DistUpgradeController.py:2119
msgid "Upgrade complete"
msgstr "Üst-qademeleme tamam"

#: ../DistUpgrade/DistUpgradeController.py:2052
#: ../DistUpgrade/DistUpgradeController.py:2103
#: ../DistUpgrade/DistUpgradeController.py:2109
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr ""
"Üst-qademeleme tamamlanğandır faqat üst-qademeleme ketişatı esnasında "
"hatalar bar edi."

# tr
#: ../DistUpgrade/DistUpgradeController.py:2071
msgid "System upgrade is complete."
msgstr "Sistem yükseltmesi tamamlandı."

# tr
#: ../DistUpgrade/DistUpgradeController.py:2120
msgid "The partial upgrade was completed."
msgstr "Kısmi yükseltme tamamlandı."

#: ../DistUpgrade/DistUpgradeController.py:2148
msgid "Lock screen disabled"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2149
msgid ""
"Your lock screen has been disabled and will remain disabled until you reboot."
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeFetcher.py:107
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:170
msgid "Could not find the release notes"
msgstr "Yayım notları bulunamadı"

# tr
#: ../DistUpgrade/DistUpgradeFetcher.py:108
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:171
msgid "The server may be overloaded. "
msgstr "Sunucu aşırı yüklenmiş olabilir. "

# tr
#: ../DistUpgrade/DistUpgradeFetcher.py:120
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:175
msgid "Could not download the release notes"
msgstr "Yayım notları indirilemedi"

# tr
#: ../DistUpgrade/DistUpgradeFetcher.py:121
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
msgid "Please check your internet connection."
msgstr "Lütfen internet bağlantınızı denetleyin."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:73
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:97
#, python-format
msgid "extracting '%s'"
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeFetcherCore.py:118
#: ../DistUpgrade/DistUpgradeFetcherCore.py:119
msgid "Could not run the upgrade tool"
msgstr "Yükseltme aracı çalıştırılamadı."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:120
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeFetcherCore.py:197
msgid "Upgrade tool signature"
msgstr "Yükseltme aracı imzası"

# tr
#: ../DistUpgrade/DistUpgradeFetcherCore.py:204
msgid "Upgrade tool"
msgstr "Yükseltme aracı"

# tr
#: ../DistUpgrade/DistUpgradeFetcherCore.py:238
msgid "Failed to fetch"
msgstr "Getirme başarısız"

# tr
#: ../DistUpgrade/DistUpgradeFetcherCore.py:239
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr "Yükseltme indirilemedi. Ağda bir sorun olabilir. "

# tr
#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
msgid "Authentication failed"
msgstr "Kimlik denetimi başarısız oldu"

# tr
#: ../DistUpgrade/DistUpgradeFetcherCore.py:244
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Yükseltme için kimlik doğrulama başarısız oldu. Ağda veya sunucuda bir sorun "
"olabilir. "

# tr
#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
msgid "Failed to extract"
msgstr "Çıkarılamadı"

# tr
#: ../DistUpgrade/DistUpgradeFetcherCore.py:250
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr "Yükseltme açılamadı. Ağda ya da sunucunuda bir sorun olabilir. "

# tr
#: ../DistUpgrade/DistUpgradeFetcherCore.py:255
msgid "Verification failed"
msgstr "Doğrulama başarısız oldu"

# tr
#: ../DistUpgrade/DistUpgradeFetcherCore.py:256
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr "Yükseltme onaylanamadı. Ağda ya da sunucuda bir sorun olabilir. "

# tr
#: ../DistUpgrade/DistUpgradeFetcherCore.py:270
#: ../DistUpgrade/DistUpgradeFetcherCore.py:276
msgid "Can not run the upgrade"
msgstr "Yükseltme çalıştırılamıyor"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:271
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeFetcherCore.py:277
#, python-format
msgid "The error message is '%s'."
msgstr "Hata mesajı '%s'"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:151
msgid "&Upgrade"
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:154
#: ../DistUpgrade/DistUpgradeViewKDE.py:715
msgid "&Cancel"
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:156
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "Çıqarılış Notları"

# tr
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:191
msgid "Upgrade"
msgstr "Yükselt"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:207
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:222
msgid "Downloading additional package files..."
msgstr "Ek paket dosyeleri endirile..."

# tr
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:224
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Dosya %s / %s, %sB/s hızla"

# tr
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:228
#, python-format
msgid "File %s of %s"
msgstr "Dosya %s / %s"

# tr
#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:235
#: ../DistUpgrade/DistUpgradeViewGtk3.py:125
#: ../DistUpgrade/DistUpgradeViewKDE.py:302
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Lütfen \"%2s\" sürücüsüne \"%1s\" takın"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:237
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:242
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:243
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
msgid "Media Change"
msgstr "Vasat Deñişimi"

#: ../DistUpgrade/DistUpgradeQuirks.py:219
msgid "Your graphics hardware may not be fully supported in Ubuntu 14.04."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:221
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D Do you still want to continue with the "
"upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:245
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:247
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeQuirks.py:267
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Yükseltme, masaüstü efektlerini, oyunların ve diğer yoğun grafik kullanan "
"uygulamaların başarımını düşürebilir."

# tr
#: ../DistUpgrade/DistUpgradeQuirks.py:271
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Bu bilgisayar al-azırda AMD 'fglrx' grafik sürücisini qullana. Bu sürüciniñ "
"Ubuntu 10.04 LTS'te donanımıñız ile çalışacaq faydalanışlı bir sürümi yoq.\n"
"\n"
"Devam etmege isteysiñizmi?"

#: ../DistUpgrade/DistUpgradeQuirks.py:310
msgid "Sorry, no more upgrades for this system"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:311
#, python-format
msgid ""
"There will not be any further Ubuntu releases for this system's 'i386' "
"architecture.\n"
"\n"
"Updates for Ubuntu %s will continue until %s."
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeQuirks.py:326
msgid "No ARMv6 CPU"
msgstr "ARMv6 İşlemci Yok"

# tr
#: ../DistUpgrade/DistUpgradeQuirks.py:327
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Sisteminiz, ARMv6 mimarisinden daha eski bir ARM işlemci kullanıyor. "
"Karmic'teki tüm paketler, en az ARMv6 mimarisini gerektirecek iyileştirmeler "
"ile oluşturulmuştur. Bu donanım ile, sisteminizi yeni bir Ubuntu sürümüne "
"yükseltmek mümkün değil."

# tr
#: ../DistUpgrade/DistUpgradeQuirks.py:347
msgid "No init available"
msgstr "Başlatıcı mevcut değil"

#: ../DistUpgrade/DistUpgradeQuirks.py:348
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Sistemiñiz başlanğıç cını olmağan sanallaştırılğan bir çevre, meselâ Linux-"
"VServer, kibi körüne. Ubuntu 10.04 bu tür çevre içerisinde çalışalmaz; evelâ "
"sanal maşnañıznıñ yapılandırılışınıñ yañartılması şarttır.\n"
"\n"
"Devam etmege istegeniñizden eminsiñizmi?"

#: ../DistUpgrade/DistUpgradeQuirks.py:461
msgid "Connection to the Snap Store failed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:462
msgid ""
"You have the package lxd installed but your system is unable to reach the "
"Snap Store. lxd is now provided via a snap and the release upgrade will fail "
"if snapd is not functional. Please make sure you're connected to the "
"Internet and update any firewall or proxy settings as needed so that you can "
"reach api.snapcraft.io. If you are an enterprise with a firewall setup you "
"may want to configure a Snap Store proxy."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:477
msgid "Connection to Snap Store failed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:478
msgid ""
"Your system does not have a connection to the Snap Store. For the best "
"upgrade experience make sure that your system can connect to api.snapcraft."
"io.\n"
"Do you still want to continue with the upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:487
msgid "Outdated snapd package"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:488
msgid ""
"Your system does not have the latest version of snapd. Please update the "
"version of snapd on your system to improve the upgrade experience.\n"
"Do you still want to continue with the upgrade?"
msgstr ""

#. now perform direct API calls to the store, requesting size
#. information for each of the snaps needing installation
#: ../DistUpgrade/DistUpgradeQuirks.py:511
msgid "Calculating snap size requirements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:542
msgid "Processing snap replacements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:548
#, python-format
msgid "refreshing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:552
#, python-format
msgid "removing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:555
#, python-format
msgid "installing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:584
msgid "PAE not enabled"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:585
msgid ""
"Your system uses a CPU that does not have PAE enabled. Ubuntu only supports "
"non-PAE systems up to Ubuntu 12.04. To upgrade to a later version of Ubuntu, "
"you must enable PAE (if this is possible) see:\n"
"http://help.ubuntu.com/community/EnablingPAE"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:895
msgid "Checking for installed snaps"
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeMain.py:58
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Yükseltilebilir paketler içeren bir cdrom'u aramak için verilen yolu kullan"

# tr
#: ../DistUpgrade/DistUpgradeMain.py:64
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Önyüzü kullan. Şu anda kullanılabilir olanlar: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:67
msgid "*DEPRECATED* this option will be ignored"
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeMain.py:70
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "Sadece kısmi yükseltme yap (source.list dosyası yeniden yazılmadan)"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid "Disable GNU screen support"
msgstr "GNU ekran destegini ğayrıqabilleştir"

# tr
#: ../DistUpgrade/DistUpgradeMain.py:75
msgid "Set datadir"
msgstr "Veri dizinini ayarla"

#: ../DistUpgrade/DistUpgradeMain.py:78
msgid "Upgrade to the development release"
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeViewGtk3.py:146
#: ../DistUpgrade/DistUpgradeViewKDE.py:316
msgid "Fetching is complete"
msgstr "Getirme tamamlandı"

# tr
#: ../DistUpgrade/DistUpgradeViewGtk3.py:161
#: ../DistUpgrade/DistUpgradeViewKDE.py:332
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "%li / %li dosya %sB/s hızla alınıyor"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:163
#: ../DistUpgrade/DistUpgradeViewGtk3.py:325
#: ../DistUpgrade/DistUpgradeViewKDE.py:333
#: ../DistUpgrade/DistUpgradeViewKDE.py:483
#, python-format
msgid "About %s remaining"
msgstr "Taqriben %s qala"

# tr
#: ../DistUpgrade/DistUpgradeViewGtk3.py:166
#: ../DistUpgrade/DistUpgradeViewKDE.py:335
#, python-format
msgid "Fetching file %li of %li"
msgstr "Dosyalar inidiriliyor. İnen: %li  Toplam: %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk3.py:198
#: ../DistUpgrade/DistUpgradeViewKDE.py:373
msgid "Applying changes"
msgstr "Deñişiklikler uyğulana"

# tr
#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk3.py:224
#: ../DistUpgrade/DistUpgradeViewKDE.py:386
msgid "dependency problems - leaving unconfigured"
msgstr "bağımlılık sorunları - yapılandırılmadan bırakılıyor."

# tr
#: ../DistUpgrade/DistUpgradeViewGtk3.py:229
#: ../DistUpgrade/DistUpgradeViewKDE.py:388
#, python-format
msgid "Could not install '%s'"
msgstr "\"%s\" kurulamadı"

# tr
#: ../DistUpgrade/DistUpgradeViewGtk3.py:230
#: ../DistUpgrade/DistUpgradeViewKDE.py:389
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Yükseltme devam edecek fakat '%s' paketi çalışır durumda olmayabilir. Lütfen "
"bununla ilgili bir hata raporlayın."

# tr
#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk3.py:247
#: ../DistUpgrade/DistUpgradeViewKDE.py:411
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Özelleştirilmiş yapılandırma dosyası\n"
"'%s' değiştirilsin mi?"

# tr
#: ../DistUpgrade/DistUpgradeViewGtk3.py:248
#: ../DistUpgrade/DistUpgradeViewKDE.py:412
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Eğer bu yapılandırma dosyasını yeni sürümüyle değiştirecekseniz bu dosyaya "
"yapmış olduğunuz değişiklikleri kaybedeceksiniz."

# tr
#: ../DistUpgrade/DistUpgradeViewGtk3.py:268
#: ../DistUpgrade/DistUpgradeViewKDE.py:435
msgid "The 'diff' command was not found"
msgstr "\"diff\" komutu bulunamadı"

# tr
#: ../DistUpgrade/DistUpgradeViewGtk3.py:535
#: ../DistUpgrade/DistUpgradeViewText.py:140
msgid "A fatal error occurred"
msgstr "Ölümcül bir hata oluştu"

# tr
#: ../DistUpgrade/DistUpgradeViewGtk3.py:536
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Lütfen bunı (eger yapmağan iseñiz) bir illet olaraq raportlañız ve "
"raportıñızda /var/log/dist-upgrade/main.log ve /var/log/dist-upgrade/apt.log "
"dosyelerini dahil etiñiz. Üst-qademeleme abortlanğandır.\n"
"Özgün menba listeñiz /etc/apt/sources.list.distUpgrade içinde saqlandı."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:566
msgid "Ctrl-c pressed"
msgstr "Ctrl-c basıldı"

# tr
#: ../DistUpgrade/DistUpgradeViewGtk3.py:567
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Bu, işlemi iptal edecektir ve sistemi bozuk bir durumda bırakabilir. Bunu "
"yapmak istediğinizden emin misiniz ?"

# tr
#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk3.py:706
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Veri kaybını önlemek için açık olan tüm uygulamaları ve belgeleri kapatın."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:720
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Canonical tarafından artıq desteklenmey (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:721
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Alt-qademele (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:722
#, python-format
msgid "Remove (%s)"
msgstr "Çetleştir (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:723
#, python-format
msgid "No longer needed (%s)"
msgstr "Artıq kerekmey (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:724
#, python-format
msgid "Install (%s)"
msgstr "Qur (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:725
#, python-format
msgid "Upgrade (%s)"
msgstr "Üst-qademele (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:773
msgid "Cancel"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:774
#: ../DistUpgrade/DistUpgradeViewKDE.py:939
msgid "Continue"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:447
msgid "Show Difference >>>"
msgstr "Farqnı Köster >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:450
msgid "<<< Hide Difference"
msgstr "<<< Farqnı Gizle"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Error"
msgstr "Hata"

#: ../DistUpgrade/DistUpgradeViewKDE.py:719
#: ../DistUpgrade/DistUpgradeViewKDE.py:960
msgid "&Close"
msgstr "&Qapat"

#: ../DistUpgrade/DistUpgradeViewKDE.py:753
msgid "Show Terminal >>>"
msgstr "Terminalnı Köster >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:756
msgid "<<< Hide Terminal"
msgstr "<<< Terminalnı Gizle"

#: ../DistUpgrade/DistUpgradeViewKDE.py:829
msgid "Information"
msgstr "Malümat"

#: ../DistUpgrade/DistUpgradeViewKDE.py:877
msgid "&Start Upgrade"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:879
#: ../DistUpgrade/DistUpgradeViewKDE.py:923
#: ../DistUpgrade/DistUpgradeViewKDE.py:926
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Tafsilât"

#: ../DistUpgrade/DistUpgradeViewKDE.py:901
#, python-format
msgid "No longer supported %s"
msgstr "Artıq desteklenmey %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:903
#, python-format
msgid "Remove %s"
msgstr "%s çetleştirilsin"

#: ../DistUpgrade/DistUpgradeViewKDE.py:905
#: ../DistUpgrade/DistUpgradeViewText.py:239
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "%s çetleştirilsin (avto qurulğan edi)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:907
#, python-format
msgid "Install %s"
msgstr "%s qurulsın"

#: ../DistUpgrade/DistUpgradeViewKDE.py:909
#, python-format
msgid "Upgrade %s"
msgstr "%s üst-qademelensin"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
#: ../DistUpgrade/DistUpgradeViewText.py:292
msgid "Restart required"
msgstr "Kene başlatuv şarttır"

# tr
#: ../DistUpgrade/DistUpgradeViewKDE.py:956
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Yükseltmeyi tamamlamak için sistemi yeniden başlatın</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:959
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "Şimdi _Kene Başlat"

# tr
#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:977
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Devam eden yükseltmeyi iptal etmek mi istiyorsunuz ?</big></b>\n"
"Yükseltmeyi iptal ederseniz sistem kullanılamaz duruma gelebilir. "
"Yükseltmeye devam etmeniz ısrarla tavsiye edilir."

#: ../DistUpgrade/DistUpgradeViewKDE.py:981
msgid "Cancel Upgrade?"
msgstr "Üst-qademelemeden Vazgeçilsinmi?"

# tr
#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li gün"

# tr
#: ../DistUpgrade/DistUpgradeView.py:84
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li saat"

# tr
#: ../DistUpgrade/DistUpgradeView.py:86
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li dakika"

# tr
#: ../DistUpgrade/DistUpgradeView.py:87
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li saniye"

# tr
#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:103
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

# tr
#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:121
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

# tr
#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:172
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Bu indirme işlemi 1Mbit DSL bağlantısıyla yaklaşık %s ve 56k bir modemle "
"yaklaşık %s sürecektir."

# tr
#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:176
#, python-format
msgid "This download will take about %s with your connection. "
msgstr ""
"Bu indirme işlemi sizin bağlantınızla yaklaşık olarak %s kadar sürecektir. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:268 ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Preparing to upgrade"
msgstr "Üst-qademelemege azırlanıla"

# tr
#: ../DistUpgrade/DistUpgradeView.py:269
msgid "Getting new software channels"
msgstr "Yeni yazılım kanalları alınıyor"

# tr
#: ../DistUpgrade/DistUpgradeView.py:270 ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Getting new packages"
msgstr "Yeni paketler alınıyor"

# tr
#: ../DistUpgrade/DistUpgradeView.py:271 ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Installing the upgrades"
msgstr "Yükseltmeler yükleniyor"

# tr
#: ../DistUpgrade/DistUpgradeView.py:272 ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Cleaning up"
msgstr "Temizlik yapılıyor"

#: ../DistUpgrade/DistUpgradeView.py:361
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d qurulğan paket artıq Canonical tarafından desteklenmey. Kene de "
"toplulıqtan destek alabilirsiñiz."

# tr
#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:370
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d paket kaldırılacak."

# tr
#: ../DistUpgrade/DistUpgradeView.py:375
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d yeni paket kurulacak."

# tr
#: ../DistUpgrade/DistUpgradeView.py:381
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d paket yükseltilecek."

# tr
#: ../DistUpgrade/DistUpgradeView.py:389
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Toplam %s indirmeniz gerekmektedir. "

#: ../DistUpgrade/DistUpgradeView.py:394
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:398
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:403
msgid "Removing the packages can take several hours. "
msgstr ""

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:408
msgid "The software on this computer is up to date."
msgstr ""

# tr
#: ../DistUpgrade/DistUpgradeView.py:409
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Sisteminiz için olası herhangi bir yükseltme yok. Yükseltme işlemi şimdi "
"iptal edilecek."

# tr
#: ../DistUpgrade/DistUpgradeView.py:425
msgid "Reboot required"
msgstr "Sistemi yeniden başlatmanız gerekiyor"

# tr
#: ../DistUpgrade/DistUpgradeView.py:426
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Yükseltme tamamlandı. Sistemi yeniden başlatmanız gerekiyor. Bunu şimdi "
"yapmak ister misiniz?"

#: ../DistUpgrade/DistUpgradeViewText.py:82
#, python-format
msgid "Progress: [%3i%%]"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewText.py:141
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Lütfen bunı bir illet olaraq raportlañız ve raportıñızda /var/log/dist-"
"upgrade/main.log ve /var/log/dist-upgrade/apt.log dosyelerini dahil etiñiz. "
"Üst-qademeleme abortlanğandır.\n"
"Özgün menba listeñiz /etc/apt/sources.list.distUpgrade içinde saqlandı."

# tüklü
#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "Aborting"
msgstr "Vazgeçile"

# tr
#: ../DistUpgrade/DistUpgradeViewText.py:171
msgid "Demoted:\n"
msgstr "Kaldırıldı:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
msgid "To continue please press [ENTER]"
msgstr "Devam etmek içün lütfen [ENTER] basıñız"

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
#: ../DistUpgrade/DistUpgradeViewText.py:261
msgid "Continue [yN] "
msgstr "Devam [eH] "

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
msgid "Details [d]"
msgstr "Tafsilât [t]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:219
#: ../DistUpgrade/DistUpgradeViewText.py:264
msgid "y"
msgstr "e"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:222
#: ../DistUpgrade/DistUpgradeViewText.py:271
msgid "n"
msgstr "h"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "d"
msgstr "t"

#: ../DistUpgrade/DistUpgradeViewText.py:230
#, python-format
msgid "No longer supported: %s\n"
msgstr "Artıq desteklenmey: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:235
#, python-format
msgid "Remove: %s\n"
msgstr "Çetleştir: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:245
#, python-format
msgid "Install: %s\n"
msgstr "Qur: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:250
#, python-format
msgid "Upgrade: %s\n"
msgstr "Üst-qademele: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:268
msgid "Continue [Yn] "
msgstr "Devam Et [eH] "

# tr
#: ../DistUpgrade/DistUpgradeViewText.py:293
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Güncellemeyi tamamlamak için yeniden başlatınız.\n"
"Eğer 'y' tuşuna basarsanız yeniden başlatılacaktır."

#: ../DistUpgrade/GtkProgress.py:71
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr ""
"Dosye %(current)li endirile (topyekün: %(total)li; sur'at: %(speed)s/s)"

# tüklü
#: ../DistUpgrade/GtkProgress.py:77
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Dosye %(current)li endirile (topyekün: %(total)li)"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "Ferdiy dosyelerniñ teraqqiyatını köster"

# tr
#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_Yükseltmeyi İptal Et"

# tr
#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "Yükseltmeye _Devam Et"

# tr
#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Yükseltme iptal edilsin mi?</big></b>\n"
"\n"
"Yükseltmeyi iptal ederseniz, sistem, kullanılamaz bir konumda kalabilir. "
"Yükseltmeyi devam ettirmeniz şiddetle önerilir."

# tr
#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Yükseltmeye Başla"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_İvaz Et"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Dosyeler arasındaki farq"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "_İlletni Maruza Et"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Devam Et"

# tr
#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Yükseltme başlatılsın mı?</big></b>"

# tr
#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Yükseltme işlemini tamamlamak için sistemi yeniden başlatın</big></"
"b>\n"
"Devam etmeden önce lütfen çalışmalarınızı kaydedin."

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Dağıtım Üst-qademelemesi"

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
#, no-c-format
msgid "<b><big>Upgrading Ubuntu to version %s</big></b>"
msgstr ""

# tr
#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "    "
msgstr "    "

# tr
#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Setting new software channels"
msgstr "Yeni yazılım kanalları ayarlanıyor"

#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Restarting the computer"
msgstr "Bilgisayar kene başlatıla"

#: ../data/gtkbuilder/DistUpgrade.ui.h:28
msgid "Terminal"
msgstr "Terminal"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "_Üst-qademele"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr ""
"<b>Ubuntu'nıñ yañı bir sürümi faydalanışlıdır. Üst-qademelemege ister "
"ediñizmi?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "Üst-qademeleme"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "Maña Soñra Sora"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "Ebet, Şimdi Üst-qademele"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Yañı Ubuntu'ğa üst-qademelemeni red ettiñiz"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr ""

# tr
#: ../do-partial-upgrade:75 ../do-release-upgrade:80
msgid "Show version and exit"
msgstr "Sürümü göster ve çık"

# tr
#: ../do-partial-upgrade:78 ../do-release-upgrade:87
msgid "Directory that contains the data files"
msgstr "Veri dosyalarını içeren dizin"

# tr
#: ../do-partial-upgrade:81 ../do-release-upgrade:100
msgid "Run the specified frontend"
msgstr "Belirlenmiş önyüzü çalıştır"

# tr
#: ../do-partial-upgrade:112
msgid "Running partial upgrade"
msgstr "_Yükseltmeye başla"

# tr
#: ../do-release-upgrade:34 ../do-release-upgrade:46
msgid "Downloading the release upgrade tool"
msgstr "Sürüm güncelleme aracı indiriliyor"

#: ../do-release-upgrade:83
msgid ""
"If using the latest supported release, upgrade to the development release"
msgstr ""

# tr
#: ../do-release-upgrade:90 ../check_new_release_gtk.py:181
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"$distro-proposed 'daki güncelleyiciyi kullanarak en son duyuruya "
"güncellemeyi deneyin"

# tr
#: ../do-release-upgrade:94
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Özel bir güncelleme kipinde çalış.\n"
"Şu anda, bir masaüstü sisteminin düzenli güncellemeleri için 'masaüstü' ve "
"sunucu sistemlerinin düzenli güncellemeleri için 'sunucu' desteklenmektedir."

# tr
#: ../do-release-upgrade:103
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Sadece eğer yeni dağıtım güncellemesi kullanılabildiğinde kontrol et ve "
"sonucu çıkış koduyla bildir."

#: ../do-release-upgrade:108
msgid ""
"Try the upgrade with third party mirrors and repositories enabled instead of "
"commenting them out."
msgstr ""

#: ../do-release-upgrade:122
msgid "The options --devel-release and --proposed are"
msgstr ""

#: ../do-release-upgrade:123
msgid "mutually exclusive. Please use only one of them."
msgstr ""

#: ../do-release-upgrade:127
msgid "Checking for a new Ubuntu release"
msgstr ""

#: ../do-release-upgrade:139
msgid "Your Ubuntu release is not supported anymore."
msgstr "Ubuntu sürümiñiz artıq desteklenmey"

#: ../do-release-upgrade:140
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Üst-qademeleme malümatı içün, lütfen ziyaret etiñiz:\n"
"%(url)s\n"

#: ../do-release-upgrade:147
msgid "In /etc/update-manager/release-upgrades Prompt "
msgstr ""

#: ../do-release-upgrade:148
msgid "is set to never so upgrading is not possible."
msgstr ""

#: ../do-release-upgrade:150
msgid "There is no development version of an LTS available."
msgstr ""

#: ../do-release-upgrade:151
msgid "To upgrade to the latest non-LTS development release "
msgstr ""

#: ../do-release-upgrade:152
msgid "set Prompt=normal in /etc/update-manager/release-upgrades."
msgstr ""

#: ../do-release-upgrade:154
msgid "Upgrades to the development release are only "
msgstr ""

#: ../do-release-upgrade:155
msgid "available from the latest supported release."
msgstr ""

#: ../do-release-upgrade:157
msgid "No new release found."
msgstr ""

#: ../do-release-upgrade:162
msgid "Release upgrade not possible right now"
msgstr "Çıqarılış üst-qademelemesi şu ande mümkün degil"

#: ../do-release-upgrade:163
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Çıqarılış üst-qademelemesi al-azırda icra etilalmay, lütfen soñra yañıdan "
"deñeñiz. Sunucı raportladı: '%s'"

# tr
#: ../do-release-upgrade:169
#, c-format
msgid "New release '%s' available."
msgstr "Yeni sürüm çıktı ('%s')."

# tr
#: ../do-release-upgrade:170
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Yükseltmek için 'do-release-upgrade' çalıştırın."

#: ../do-release-upgrade:187
msgid "Please install all available updates for your release before upgrading."
msgstr ""

#: ../do-release-upgrade:203
msgid ""
"You have not rebooted after updating a package which requires a reboot. "
"Please reboot before upgrading."
msgstr ""

#: ../check_new_release_gtk.py:85
msgid "Software Updater"
msgstr ""

# tr
#: ../check_new_release_gtk.py:113
#, python-format
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Ubuntu %(version)s Yükseltmesi Mevcut"

# tr
#. print("don't upgrade")
#: ../check_new_release_gtk.py:144
#, python-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Ubuntu %s sürümüne yükseltmeyi reddettiniz"

# tr
#: ../check_new_release_gtk.py:177
msgid "Check if upgrading to the latest devel release is possible"
msgstr "En son geliştirme sürümüne yükseltme olasılığını denetle"

#: ../check_new_release_gtk.py:186
msgid "Add debug output"
msgstr ""

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade:\n"
#~ "%s\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "Üst-qademe esaplanğanda hal etilmez bir hata meydanğa keldi:\n"
#~ "%s \n"
#~ "Sebebi aşağıdakiler olabilir:\n"
#~ " * Ubuntu'nıñ bir çıqarılışaldı sürümine üst-qademelev\n"
#~ " * Ubuntu'nıñ cari çıqarılışaldı sürümini çaptıruv\n"
#~ " * Ubuntu tarafından temin etilmegen ğayrı resmiy yazılım paketleri\n"
#~ "\n"

#~ msgid ""
#~ "Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop or edubuntu-desktop package and it was not possible to detect "
#~ "which version of Ubuntu you are running.\n"
#~ " Please install one of the packages above first using synaptic or apt-get "
#~ "before proceeding."
#~ msgstr ""
#~ "Sistemiñiz bir ubuntu-desktop, kubuntu-desktop yaki edubuntu-desktop "
#~ "paketi ihtiva etmey ve angi Ubuntu sürümini qullanğanıñız tesbit "
#~ "etilamadı.\n"
#~ " Lütfen devam etmezden evel, synaptic yaki apt-get qullanaraq yuqarıdaki "
#~ "paketlerden birini quruñız."

# tr
#~ msgid "Sandbox setup failed"
#~ msgstr "Çalışma dizini kurulumu başarısız"

# tr
#~ msgid "It was not possible to create the sandbox environment."
#~ msgstr "Çalışma dizini ortamı oluşturmak mümkün değil."

# tr
#~ msgid "Sandbox mode"
#~ msgstr "Çalışma dizini kipi"

# tr
#~ msgid ""
#~ "Your python install is corrupted. Please fix the '/usr/bin/python' "
#~ "symlink."
#~ msgstr ""
#~ "Python kurulumunuzda hata var. Lütfen '/usr/bin/python' sembolik bağını "
#~ "onarın."

# tr
#~ msgid "Package 'debsig-verify' is installed"
#~ msgstr "'debsig-verify' paketi kuruldu"

# tr
#~ msgid ""
#~ "The upgrade can not continue with that package installed.\n"
#~ "Please remove it with synaptic or 'apt-get remove debsig-verify' first "
#~ "and run the upgrade again."
#~ msgstr ""
#~ "Bu paket kuruluyken güncelleme devam edemez.\n"
#~ "Lütfen önce synaptic veya 'apt-get remove debsig-verify' ile paketi "
#~ "kaldırın ve güncellemeyi tekrar çalıştırın."

# tüklü
#~ msgid ""
#~ "While scanning your repository information no mirror entry for the "
#~ "upgrade was found. This can happen if you run a internal mirror or if the "
#~ "mirror information is out of date.\n"
#~ "\n"
#~ "Do you want to rewrite your 'sources.list' file anyway? If you choose "
#~ "'Yes' here it will update all '%s' to '%s' entries.\n"
#~ "If you select 'No' the upgrade will cancel."
#~ msgstr ""
#~ "Repo malümatıñız taranğanda üst-qademe içün bir küzgü kirişi tapılmadı. "
#~ "Bu, dahiliy bir küzgü qullansañız yaki küzgü malümatı küncel degil ise "
#~ "sudur etebilir.\n"
#~ "\n"
#~ "'sources.list' dosyeñizni kene de kene yazmağa isteysiñizmi? Eger mında "
#~ "'Ebet' saylasañız, '%s' kirişleriniñ episi '%s' olaraq yañartılacaq.\n"
#~ "Eger 'Hayır' saylasañız üst-qademeden vazgeçilecek."

#~ msgid ""
#~ "The upgrade has aborted. The upgrade needs a total of %s free space on "
#~ "disk '%s'. Please free at least an additional %s of disk space on '%s'. "
#~ "Empty your trash and remove temporary packages of former installations "
#~ "using 'sudo apt-get clean'."
#~ msgstr ""
#~ "Üst-qademeleme abortlandı. Üst-qademeleme '%s' diski üzerinde topyekün %s "
#~ "boş fezağa muhtacdır. Lütfen '%s' diski üzerinde eñ az ek %s fezanı azat "
#~ "etiñiz. Çöpüñizni temizleñiz ve 'sudo apt-get clean' qullanaraq evelki "
#~ "qurulumlarnıñ muvaqqat paketlerini siliñiz."

# tr
#~ msgid "evms in use"
#~ msgstr "evms kullanımda"

# tr
#~ msgid ""
#~ "Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
#~ "software is no longer supported, please switch it off and run the upgrade "
#~ "again when this is done."
#~ msgstr ""
#~ "Sisteminiz /proc/mounts içindeki 'evms' birim yöneticisini kullanıyor. "
#~ "'evms' yazılımı artık desteklenmiyor, lütfen 'evms'yi kapattıktan sonra, "
#~ "bu bittiğinde yükseltmeyi yeniden çalıştırın."

#~ msgid ""
#~ "This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
#~ "version of this driver is available that works with your video card in "
#~ "Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "Bu bilgisayar al-azırda NVIDIA 'nvidia' grafik sürücisini qullana. Bu "
#~ "sürüciniñ Ubuntu 10.04 LTS'te video kartıñız ile çalışacaq faydalanışlı "
#~ "bir sürümi yoq.\n"
#~ "\n"
#~ "Devam etmege isteysiñizmi?"

# tüklü
#~ msgid "No i686 CPU"
#~ msgstr "i686 prosessorı yoq"

# tr
#~ msgid ""
#~ "Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
#~ "extension. All packages were built with optimizations requiring i686 as "
#~ "the minimal architecture. It is not possible to upgrade your system to a "
#~ "new Ubuntu release with this hardware."
#~ msgstr ""
#~ "Sistemiñiz, bir i586 prosessorını yaki 'cmov' uzantısı olmağan bir "
#~ "prosessornı qullana. Paketlerniñ episi asğariy olaraq i686 mimarlığını "
#~ "zarur etken optimalleştirmeler ile inşa etilgendir. Bu donanım ile, "
#~ "sistemiñizni yañı bir Ubuntu çıqarılışına üst-qademelemek mümkün degildir."

# tr
#~ msgid "Sandbox upgrade using aufs"
#~ msgstr "Çalışma dizini yükseltmesi 'aufs' kullanıyor"

# tr
#~ msgid "Test upgrade with a sandbox aufs overlay"
#~ msgstr "Yükseltmeyi, bir çalışma dizini 'aufs' yer paylaşımıyla sına"

# tr
#~ msgid "No new release found"
#~ msgstr "Yeni sürüm bulunamadı"
